drivers/hmis/lib/form_data/default/fragments/signature.json

```diff
@@ -0,0 +1,100 @@
+{
+  "type": "GROUP",
+  "variant": "SIGNATURE",
+  "link_id": "signature_group",
+  "text": "eSignature",
+  "subtitle": "The following signatures are required on this document.",
+  "item": [
+    {
+      "type": "GROUP",
+      "text": "Head of Household",
+      "link_id": "head_of_household_signature_group",
+      "item": [
+        {
+          "type": "STRING",
+          "link_id": "head_of_household_signature",
+          "text": "Full Name",
+          "required": true,
+          "mapping": {
+            "custom_field_key": "head_of_household_signature"
+          }
+        },
+        {
+          "type": "DATE",
+          "link_id": "head_of_household_signature_date",
+          "text": "Today's Date",
+          "required": true,
```

Comment:
Maybe we should use:
```json
 "disabled_display": "PROTECTED"
```
This always shows the date field but it is disabled if there's no signature